map/src/components/resultDetail.tsx

```diff
@@ -0,0 +1,247 @@
+    line-height: 16px;
+  }
+
+  > .source {
+    font-size: 12px;
+    line-height: 150%;
+  }
+
+  > .name {
+    font-size: 1.5rem;
+    padding-bottom: ${p => p.theme.spacingPx / 2}px;
+  }
+
+  > .location {
+    color: ${p => p.theme.textColorLight};
+    font-size: 0.9rem;
+  }
+
+  > .marker-type {
+    margin-bottom: ${p => p.theme.spacingPx / 2}px;
+  }
+
+  > .contact-group {
+    margin-top: ${p => p.theme.spacingPx}px;
+    color: ${p => p.theme.textColorLight};
+
+    ul {
+      padding-left: ${p => p.theme.spacingPx}px;
+      margin-left: ${p => p.theme.spacingPx}px;
+      a {
+        display: inline-flex;
+        align-items: center;
+        text-decoration: none;
+
+        &:hover {
+          text-decoration: underline;
+        }
+
+        svg {
+          margin-right: ${p => p.theme.spacingPx / 2}px;
+        }
+      }
+    }
+  }
+`;
+
+const SOURCES = {
+  'mutualaid.wiki': {
+    label: 'Mutual Aid Wiki',
+    href: 'https://mutualaid.wiki/',
+  },
+  'mutualaidhub.org': {
+    label: 'Mutual Aid Hub',
+    href: 'https://www.mutualaidhub.org/',
+  },
+  hardcoded: null,
+};
+
+const contactInfo = (lang: Language, label: string, info?: ContactDetails) => {
+  if (!info) {
+    return null;
+  }
+  const items: Array<JSX.Element> = [];
+  if (info.phone) {
+    items.push(
+      ...info.phone.map(number => (
+        <a key={items.length} href={`tel:${number.replace(/\s/g, '')}`}>
+          <MdPhone />
+          {number}
+        </a>
+      )),
+    );
+  }
+  if (info.email) {
+    items.push(
+      ...info.email.map(email => (
+        <a
+          key={items.length}
+          href={`mailto:${email}`}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <MdEmail />
+          {email}
+        </a>
+      )),
+    );
+  }
+  if (info.facebookGroup) {
+    items.push(
+      <a
+        key={items.length}
+        href={info.facebookGroup}
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        <MdLanguage />
+        {t(lang, s => s.results.contact.facebookGroup)}
+      </a>,
+    );
+  }
+  if (info.web) {
+    items.push(
+      ...Object.entries(info.web).map(entry => (
+        <a
+          key={items.length}
+          href={entry[1]}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <MdLanguage />
+          {entry[0]}
+        </a>
+      )),
+    );
+  }
+  if (items.length === 0) {
+    return null;
+  }
+  return (
+    <div className="contact-group">
+      <strong>{label}</strong>
+      <ul>
+        {items.map((item, i) => (
+          <li key={i}>{item}</li>
+        ))}
+      </ul>
+    </div>
+  );
+};
+
+const ResultDetail = ({ result, lang, fullDetail = false }: Props) => {
+  const selectedResultSource =
+    result?.info.source && SOURCES[result.info.source.name];
+  const selectedResultSentenceType =
+    result?.info.type.type === 'mutual-aid-group' ||
+    result?.info.type.type === 'org'
```

Comment:
Why mutual aid group and org?

Reply:
@jensren this code was there already too, I wasn't really sure about this either..